qa/cached_inventory.py

```diff
@@ -0,0 +1,119 @@
+import requests
+import json
+from datetime import datetime
+from collections import OrderedDict
+from test_framework.test_framework import OpenBazaarTestFramework, TestFailure
+
+
+class CachedInventoryTest(OpenBazaarTestFramework):
+
+    def __init__(self):
+        super().__init__()
+        self.num_nodes = 2
+
+    def setup_network(self):
+        self.setup_nodes()
+
+    def run_test(self):
+        alice = self.nodes[0]
+        bob = self.nodes[1]
```

Comment:
Can we call them by their role in the test instead of an arbitrary name?

Reply:
@placer14 I understand that when there are real "roles" like vendor, buyer, moderator, etc, but in this case they're just two nodes that are both vendors and are looking at each other's inventories. What role names would you suggest? The best I can think of is just "node1" and "node2" but I think that's even less clear.